index.html

```diff
@@ -13,20 +13,20 @@
   </head>
   <body>
     <header>
-      <section id="session">
-        <a class="sess-link" href="#"><i class="sess-icon"><img class="face" src="images/icons/facebook-icon.png" alt="facebook icon"></i></a>
-        <a class="sess-link" href="#"><i class="sess-icon"><img class="twit" src="images/icons/twitter-icon.png" alt="twitter icon"></i></a>
-        <a class="sess-link" href="#"><p class="sess-icon">English</p></a>
-        <a class="sess-link" href="#"><p class="sess-icon">MyPage</p></a>
-        <a class="sess-link" href="#"><p class="sess-icon">Logout</p></a>
-      </section>
+      <ul id="session">
+        <li href="#"><a class="sess-link"><i><img class="face" src="./images/icons/facebook-icon.png" alt="facebook icon"></i></a></li>
+        <li href="#"><a class="sess-link"><i><img class="twit" src="./images/icons/twitter-icon.png" alt="twitter icon"></i></a></li>
+        <li href="#"><a class="sess-link">English</a></li>
+        <li href="#"><a class="sess-link">MyPage</a></li>
```

Comment:
As requested by the previous reviewer, kindly fix the overflow issues. The below screenshot shows the reason for the overflow. Kindly fix it to improve user experience on the website across screen sizes.

![image](https://user-images.githubusercontent.com/20554583/218238114-1fa5a1b2-9521-4ed4-8e90-6fa05c40ffc2.png)

The website on 1588px screen:
![image](https://user-images.githubusercontent.com/20554583/218238141-20aca700-8e6f-4928-91aa-f204d227f3f9.png)

Reply:
Oh, I see! Thanks!!!